packages/manager/modules/pci/src/projects/project/quota/quota.html

```diff
@@ -54,21 +54,11 @@
             <a
                 class="oui-button oui-button_primary oui-link_icon"
                 data-ng-if=":: $ctrl.quotas.length > 0"
-                data-ng-href="{{:: $ctrl.supportUrl }}"
-                rel="noopener"
-                target="_blank"
+                data-ui-sref="pci.projects.project.quota.increase"
```

Comment:
We should avoid using `ui-sref` in views to abstract from ui router and prefer inject this through `resolve` if possible